Source/WebKit/UIProcess/WebAuthentication/fido/CtapHidDriver.cpp

```diff
@@ -62,8 +62,9 @@ void CtapHidDriverWorker::transact(fido::FidoHidMessage&& requestMessage, Messag
     m_callback = WTFMove(callback);
 
     // HidConnection could hold data from other applications, and thereofore invalidate it before each transaction.
-    m_connection->invalidateCache();
-    m_connection->send(m_requestMessage->popNextPacket(), [weakThis = WeakPtr { *this }](HidConnection::DataSent sent) mutable {
+    Ref connection = m_connection;
+    connection->invalidateCache();
+    connection->send(m_requestMessage->popNextPacket(), [weakThis = WeakPtr { *this }](HidConnection::DataSent sent) mutable {
         ASSERT(RunLoop::isMain());
         if (!weakThis)
```

Comment:
Isn't `this` RefCounted`? If so, I'd expect us to convert the weakThis into a protectedThis RefPtr here.

Reply:
Not yet. That's the task of next patch.

Reply:
Yes, I'm actually working on it :D